__tests__/index.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const lc39 = require('@mia-platform/lc39')
+import { FastifyInstance } from 'fastify'
+
+
+let fastify: FastifyInstance
```

Comment:
This variable, the `beforeAll` and `afterAll` functions should be put inside the `describe` test suite

Reply:
Done